### PR TITLE
Problematic terminology

### PR DESCRIPTION
**Pull request by tomponline**
Replaces problematic terminology.

### CODE REVIEW

**Comment by stgraber** [comment]
We should live `lxd_bpf.h` alone as it's a verbatim copy of a system file AFAICT.

File: lxd/instance/drivers/driver_qemu.go [hunk 1]
```diff
@@ -2512,7 +2512,7 @@ func (d *qemu) addRootDriveConfig(sb *strings.Builder, mountInfo *storagePools.M
 	}
 
 	// If the storage pool is on ZFS and backed by a loop file and we can't use DirectIO, then resort to
-	// unsafe async I/O to avoid kernel hangs when running ZFS storage pools in an image file on another FS.
+	// unsafe async I/O to avoid kernel crash when running ZFS storage pools in an image file on another FS.
```
Comment (stgraber) [inline lxd/instance/drivers/driver_qemu.go:2515]: lock up would probably be better here as I associate a kernel crash with a panic

File: lxd/instance/drivers/driver_qemu.go [hunk 2]
```diff
@@ -2622,7 +2622,7 @@ func (d *qemu) addDriveConfig(sb *strings.Builder, bootIndexes map[string]int, d
 		}
 
 		// If backing FS is ZFS or BTRFS, avoid using direct I/O and use host page cache only.
-		// We've seen ZFS hangs and BTRFS checksum issues when using direct I/O on image files.
+		// We've seen ZFS crashes and BTRFS checksum issues when using direct I/O on image files.
```
Comment (stgraber) [inline lxd/instance/drivers/driver_qemu.go:2625]: same, this isn't a crash

File: test/suites/basic.sh [hunk 3]
```diff
@@ -337,12 +337,12 @@ test_basic_usage() {
   # Create and start a container
   lxc launch testimage foo
   lxc list | grep foo | grep RUNNING
-  lxc stop foo --force  # stop is hanging
+  lxc stop foo --force  # stop is blocking
```
Comment (stgraber) [inline test/suites/basic.sh:340]: I'd just drop the comment entirely as it's a common thing throughout the testsuite.

**Comment by brauner** [comment]
> We should leave `lxd_bpf.h` alone as it's a verbatim copy of a system file AFAICT.

Yes!

**Comment by brauner** [comment]
> Replaces problematic terminology.

Ah yeah, I converted all our public facing terminology in both LXD and LXC a while back. For example in:

```
commit 0c987f66b6f1cb7b59709365aa2f20bb8c3421d1
Author: Christian Brauner <christian.brauner@ubuntu.com>
Date:   Fri Jul 3 15:33:13 2020 +0200

    instance: introduce container_syscall_filtering_allow_deny extension

    Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>
```

Seems we have a few more. I think I've gotten rid of all the "slave"/"master" terminology too.

**Comment by tomponline** [comment]
> > Replaces problematic terminology.
> 
> Ah yeah, I converted all our public facing terminology in both LXD and LXC a while back. For example in:
> 
> ```
> commit 0c987f66b6f1cb7b59709365aa2f20bb8c3421d1
> Author: Christian Brauner <christian.brauner@ubuntu.com>
> Date:   Fri Jul 3 15:33:13 2020 +0200
> 
>     instance: introduce container_syscall_filtering_allow_deny extension
> 
>     Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>
> ```
> 
> Seems we have a few more. I think I've gotten rid of all the "slave"/"master" terminology too.

Yeah I left the syscall stuff alone as that seemed to be an API that we should not/cannot change?

**Comment by brauner** [comment]
> > > Replaces problematic terminology.
> > 
> > 
> > Ah yeah, I converted all our public facing terminology in both LXD and LXC a while back. For example in:
> > ```
> > commit 0c987f66b6f1cb7b59709365aa2f20bb8c3421d1
> > Author: Christian Brauner <christian.brauner@ubuntu.com>
> > Date:   Fri Jul 3 15:33:13 2020 +0200
> > 
> >     instance: introduce container_syscall_filtering_allow_deny extension
> > 
> >     Signed-off-by: Christian Brauner <christian.brauner@ubuntu.com>
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Seems we have a few more. I think I've gotten rid of all the "slave"/"master" terminology too.
> 
> Yeah I left the syscall stuff alone as that seemed to be an API that we should not/cannot change?

I did it backwards compatible, i.e. I introduced the new set of options but left the old options working but just removed the old ones from the public docs.

**Comment by stgraber** [comment]
@tomponline I don't like that we just drop the `Sanity check` comments. I was using those to denote cheap early validation that's run before anything expensive is done. So I could usually find that block by looking at the comment until whatever the next comment is.

So we should replace that with `Quick checks`, `Request validation`, `Early validation` or something along those lines but not just drop them.

**Comment by tomponline** [comment]
> @tomponline I don't like that we just drop the `Sanity check` comments. I was using those to denote cheap early validation that's run before anything expensive is done. So I could usually find that block by looking at the comment until whatever the next comment is.
> 
> So we should replace that with `Quick checks`, `Request validation`, `Early validation` or something along those lines but not just drop them.

Ah OK will do, I thought that they looked superfluous, but that makes sense now.

**Comment by tomponline** [comment]
@stgraber fixed

**Comment by tomponline** [comment]
jenkins: test this please

**Pull request merged**